scripts/user_retirement/tests/test_retire_one_learner.py

```diff
@@ -19,6 +21,13 @@
 from scripts.user_retirement.utils.exception import HttpDoesNotExistException
 
 
+@override_settings(EXTRA_SERVICES_TO_RETIRE_FROM=[
+    {
+        'name': 'MOCK_SERVICE',
+        'service_base_url': 'http://fake_service_base_url',
+        'retirement_url_path': 'fake_retirement_url_path'
+    }
+])
```

Comment:
I would have expected that these settings would be overridden for a specific (new) test case related to the new functionality, not a utility function used by all test cases in this suite.

I feel like I'm missing context for why it's required here, especially where I don't see any new test cases added to this test suite.

Reply:
I noticed that all of the tests fail with the same error of not being able to find the new setting as all of the tests seem to run in to the helpers.py file at some point. 
I wanted to resolve the existing tests that were failing before creating a new test case for the added implementation.